### ISSUE TITLE
Having trouble building due to libggit2 http_parser missing symbols

### ISSUE THREAD

**Issue by aufflick** [ssp/gitx#7]
Hi, thanks for your fork!

I found it while looking at the same build issue in the old rowanj repo. Basically libgit2.a is built without http_parser.c which means http.c has unresolved symbols (eg. _http_parser_init).

I get a link error with your build script too. Have you seen this before?

Thanks.

**Comment by ssp** [comment]
@aufflick: sorry to hear that, I haven’t seen this before.

Which OS X/Xcode version are you using? Personally I have only built the application on the current X.11 / 7.3. As far as I can tell GitX only uses libgit2 as a dependency and I don’t really know how it is built (or why).

**Comment by ssp** [comment]
This issue may sound similar to what you are reporting @aufflick:

https://github.com/libgit2/libgit2/issues/3099

Perhaps its solution can help with the issue you are seeing?

**Comment by aufflick** [comment]
No, sadly it's not a linking problem per se, it's a problem compiling libgit2 (which is triggered in the Objective-git bootstrap script in this case) where one (or more) C files aren't compiled. The problem is only discovered at link time since the header file is there at compile time. I'll try updating the libgit submodule in GitX.

**Comment by larsblumberg** [comment]
Don't know if it's of interest but I just run the build script https://gist.github.com/ssp/1c152c9a70bace500108ab9bae69624f with success with latest releases of OS X and Xcode.
